homeassistant/components/conversation/__init__.py

```diff
@@ -266,15 +266,18 @@ async def websocket_list_agents(
 
     for agent_info in manager.async_get_agent_info():
         agent = await manager.async_get_agent(agent_info.id)
+
+        supported_languages = agent.supported_languages
```

Comment:
Why include `MATCH_ALL`? Can't we not include `supported_languages` when it is match all? then it will also work with current frontend.